meta-mel/fsl-arm/recipes-bsp/u-boot/u-boot-ls1_%.bbappend

```diff
@@ -0,0 +1,2 @@
+UBOOT_CONFIG = "sdcard"
```

Comment:
I'm fairly sure doing this in the recipe is wrong. This is a configuration variable which is generally set in the machine, not the recipe. I think we should use a local.conf fragment to add `UBOOT_CONFIG_ls1021atwr = "sdcard"`, the same way we do for other similar configuration changes for upstream machines like minnow (see [meta-mel/conf/local.conf.append.minnow]()).